pkg/syncer/spec/spec_process.go

```diff
@@ -357,3 +387,15 @@ func getTransformedName(syncedObject *unstructured.Unstructured) string {
 	}
 	return syncedObject.GetName()
 }
+
+func (c *Controller) getDownstreamNamespace(namespaceName string) (*unstructured.Unstructured, error) {
```

Comment:
I don't see what this func add to a direct informer lookup.

Reply:
done